include/deal.II/base/bounding_box.h

```diff
@@ -185,7 +185,9 @@ class BoundingBox
    * Return true if the point is inside the Bounding Box, false otherwise.
    */
   bool
-  point_inside(const Point<spacedim, Number> &p) const;
+  point_inside(
+    const Point<spacedim, Number> &p,
+    const double tolerance = std::numeric_limits<Number>::epsilon()) const;
```

Comment:
Can you document what the tolerance is, and maybe also link to the corresponding parameter in `GeometryInfo::is_inside_unit_cell()`.

Reply:
I added docu and improved the implementation. In the previous implementation, the reference length scale multiplied by machine eps might be zero. The reference length scale should instead always be larger than zero and of the order of the dimensions of the bounding box, i.e.,  abs(right-left). Since this is irrelevant for the unit cell with left=0, right=1, I did not refer to `is_inside_unit_cell()`.